eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelay.scala

```diff
@@ -207,7 +207,15 @@ class ChannelRelay private(nodeParams: NodeParams,
         candidateChannels
           .map { case (shortChannelId, channelInfo) =>
             val relayResult = relayOrFail(Some(channelInfo.channelUpdate))
-            context.log.debug(s"candidate channel: shortChannelId={} balanceMsat={} capacitySat={} channelUpdate={} relayResult={}", shortChannelId, channelInfo.commitments.availableBalanceForSend, channelInfo.commitments.capacity, channelInfo.channelUpdate, relayResult)
+            context.log.debug(s"candidate channel: shortChannelId=$shortChannelId availableForSend={} capacity={} channelUpdate={} result={}",
+              channelInfo.commitments.availableBalanceForSend,
+              channelInfo.commitments.capacity,
+              channelInfo.channelUpdate,
+              relayResult match {
+                case _: RelaySuccess => "success"
+                case RelayFailure(CMD_FAIL_HTLC(_, Right(failure), _, _)) => failure
```

Comment:
nit: would calling the `failure` variable `failureReason` be clearer?

Reply:
Done with c7ca7b075.